common/test/acceptance/tests/studio/test_studio_course_team.py

```diff
@@ -169,6 +169,7 @@ def test_admins_can_delete_other_users(self):
         self.page.add_user_to_course(self.other_user.get('email'))
         self._assert_user_present(self.other_user, present=True)
         self.page.delete_user_from_course(self.other_user.get('email'))
+        self.page.wait_for_page()
```

Comment:
I'd recommend moving the wait_for_page call into delete_user_from_course so other callers of the method will automatically get the fix.

Otherwise, :+1: